Dockerfile

```diff
@@ -14,18 +14,18 @@
 # limitations under the License.
 
 ## Build zend
-FROM ubuntu:18.04 as zend-builder
+FROM ubuntu:20.04 as zend-builder
 
 MAINTAINER cronic@horizen.io
 
 SHELL ["/bin/bash", "-c"]
 
-# Latest release zen 3.2.0
-ARG ZEN_COMMITTISH=v3.2.0
+ARG ZEN_COMMITTISH
 ARG IS_RELEASE=false
 # cronic <cronic@zensystem.io> https://keys.openpgp.org/vks/v1/by-fingerprint/219F55740BBF7A1CE368BA45FB7053CE4991B669
 # Luigi Varriale <luigi@horizenlabs.io> https://keys.openpgp.org/vks/v1/by-fingerprint/FC3388A460ACFAB04E8328C07BB2A1D2CFDFCD2C
-ARG MAINTAINER_KEYS="219F55740BBF7A1CE368BA45FB7053CE4991B669 FC3388A460ACFAB04E8328C07BB2A1D2CFDFCD2C"
+# otoumas <tomas@horizenlabs.io> https://keys.openpgp.org/vks/v1/by-fingerprint/2BBE2AA1A641F6147B58450BE3527B60DAACA1D8
```

Comment:
too much info in the comments = security risk?

Reply:
well these are our public keys, so I don't think it is a great risk